doc/src/quickstart/index.rst

```diff
@@ -73,6 +65,102 @@ from the VTR root directory (hereafter referred to as :term:`$VTR_ROOT`) to buil
+
+.. code-block:: bash
+
+    export VTR_FLOW_DIR=~/vtr_work/quickstart/blink_run_flow
+
+Alternatively, you can manually replace ``$VTR_FLOW_DIR`` with your preferred directory path in the commands below.
+
+Create the working directory and navigate into it:
+
+.. code-block:: bash
+
+    > mkdir -p $VTR_FLOW_DIR
+    > cd $VTR_FLOW_DIR
+
+Now lets run the script (``$VTR_ROOT/vtr_flow/scripts/run_vtr_flow.py``) passing in:
+
+* The circuit verilog file (``$VTR_ROOT/doc/src/quickstart/blink.v``)
+* The FPGA architecture file (``$VTR_ROOT/vtr_flow/arch/timing/EArch.xml``)
+
+and also specifying the options:
+
+* ``--route_chan_width 100`` a fixed FPGA routing architecture channel width.
+
+
+The resulting command is:
+
+.. code-block:: bash
+
+    > $VTR_ROOT/vtr_flow/scripts/run_vtr_flow.py \
+        $VTR_ROOT/doc/src/quickstart/blink.v \
+        $VTR_ROOT/vtr_flow/arch/timing/EArch.xml \
+        --route_chan_width 100
+
+.. note:: Options unrecognized by run_vtr_flow (like ``--route_chan_width``) are passed on to VPR.
+
+which should produce output similar to::
+
+    EArch/blink             OK     (took 0.26 seconds, overall memory peak 63.71 MiB consumed by vpr run)
+
+There are also multiple log files (including for ABC, Parmys and VPR), which by convention the script names with the ``.out`` suffix:
+
+.. code-block:: bash
+
+    > ls $VTR_FLOW_DIR/temp/*.out
+
+    0_blackboxing_latch.out  parmys.out        report_clocks.abc.out  vanilla_restore_clocks.out
+    abc0.out                 report_clk.out  restore_latch0.out     vpr.out
+
+With the main log files of interest including the Parmys log file (``parmys.out``), log files produced by ABC (e.g. ``abc0.out``), and the VPR log file (``vpr.out``).
+
+.. note::
+
+    ABC may be invoked multiple times if a circuit has multiple clock domains, producing multiple log files (``abc0.out``, ``abc1.out``, ...)
+    
+
+You will also see there are several BLIF files produced:
+
+.. code-block:: bash
+
+    > ls $VTR_FLOW_DIR/temp/*.blif
+
+    0_blink.abc.blif   0_blink.raw.abc.blif  blink.parmys.blif
+    0_blink.parmys.blif  blink.abc.blif        blink.pre-vpr.blif
+
+With the main files of interest being ``blink.parmys.blif`` (netlist produced by Parmys), ``blink.abc.blif`` (final netlist produced by ABC after clock restoration), ``blink.pre-vpr.blif`` netlist used by VPR (usually identical to ``blink.abc.blif``).
+
+Like before, we can also see the implementation files generated by VPR:
+
+.. code-block:: bash
+
+    > ls $VTR_FLOW_DIR/temp/*.net $VTR_FLOW_DIR/temp/*.place $VTR_FLOW_DIR/temp/*.route
+
+    blink.net  blink.place  blink.route
+
+which we can visualize with:
+
+.. code-block:: bash
+
+    > $VTR_ROOT/vpr/vpr \
+        $VTR_ROOT/vtr_flow/arch/timing/EArch.xml \
+        blink --circuit_file $VTR_FLOW_DIR/temp/blink.pre-vpr.blif \
+        --route_chan_width 100 \
+        --analysis --disp on
+
 
 Running VPR
```

Comment:
I think we should rename this to "Running VPR Manually"
and then add a first sentence after the heading that says
"Sometimes you may wish to run only the vpr (placement, routing and timing analysis) parts of the flow rather than the full VTR flow (which includes synthesis). To show how to do this, let's now try taking ..."